NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -87,8 +98,13 @@ def begin(
         
         # initialise sky model
         if skymodel == 'lfmap':
-            sky_model = LFmap()
-            print("Using LFmap as sky model")
+            try:
+                sky_model = LFmap()
+                print("Using LFmap as sky model")
+            except:
+                sky_model = GlobalSkyModel(freq_unit="MHz")
+                print(f"{skymodel} import not found. Defaulting to GSM2008 as sky model.")
+
         if skymodel == 'lfss':
```

Comment:
Also, if there is a reason for not using `elif`s here? This speeds up performance (currently every if statement is checked, but this is unnecessary I believe)